### PR TITLE
Generic Checkout : US Ts&Cs/Disclaimer copy+font changes (all Tiers)

### PR DESCRIPTION
**Pull request by paul-daniel-dempsey**
## What are you doing in this PR?

For US Region Only ->
- Add Tier1,2,3 specific Ts&Cs Summary component
- Modify Tier1,2 Specific Ts&Cs & Fonts

Applies to ->
1. Generic Checkout (Tier1,Tier2,Tier3 - Summary component only)
2. Contribute Checkout (Tier2/S+)
3. GW Checkout (Tier3 - Summary component only)

Also requested remove the 'Support another Way' for US, ALL checkouts above plus (Contribute Checkout- OneTime)

[Trello](https://trello.com/c/b6KrHksg/982-disclaimer-copyfont-changes-to-us-checkout-page-all-tiers-dev-work)
[Trello](https://trello.com/c/Ge30S8ZE/1006-remove-this-copy-from-the-us-checkout-page-2)

[Figma](https://www.figma.com/design/8X3c92gky4dhzscpRlO1kE/Checkout---small-amends---2024?node-id=3-13&t=ieTY1z8HWAV9AxPi-1)

## How to test

From Landing Page->
https://support.thegulocal.com/us/contribute
https://support.thegulocal.com/us/contribute#ab-tierTwoFromApi=variant
https://support.thegulocal.com/us/contribute#ab-tierThreeFromApi=variant

Direct->
https://support.thegulocal.com/us/checkout?product=Contribution&ratePlan=Monthly&price=5
https://support.thegulocal.com/us/checkout?product=SupporterPlus&ratePlan=Monthly&price=15
https://support.thegulocal.com/us/checkout?product=TierThree&ratePlan=DomesticMonthly&promoCode=TIER3_US_MONTHLY

## Screenshots

|Tier1|Tier2|Tier3|
|-----|-----|-----|
|image|image|image|

### CODE REVIEW

File: support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx [hunk 1]
```diff
@@ -4,10 +4,16 @@ import { FormSection } from 'components/checkoutForm/checkoutForm';
...
+import {
+	privacyLink,
+	subscriptionsTermsLinks,
+	supporterPlusTermsLink,
+	tier3TermsLink,
+} from 'helpers/legal';
 import { ManageMyAccountLink } from '../../pages/supporter-plus-landing/components/manageMyAccountLink';
 
-const threeTierTerms = css`
+const termsTier3 = css`
```
Comment (jamesgorrie) [inline support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx:16]:
```suggestion
const tierThreeTerms = css`
```

if following convention

File: support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx [hunk 2]
```diff
@@ -20,46 +26,106 @@ const threeTierTerms = css`
 	}
 `;
 
+export const productNameUSTier3 = 'Guardian Weekly and All-Access Digital';
```
Comment (jamesgorrie) [inline support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx:29]: Could / should we use `Three` over `3`. By following convention we'll make find/replace refactoring easier for future folks.

Comment (jamesgorrie) [reply support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx:29]: Maybe we could drop this value into the `productCatalog` file?

File: support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx [hunk 3]
```diff
@@ -20,46 +26,106 @@ const threeTierTerms = css`
...
 	paymentMethod?: PaymentMethod;
+	countryGroupId?: CountryGroupId;
 }): JSX.Element {
-	const threeTierTermsLink =
-		'https://www.theguardian.com/info/article/2024/jul/19/digital-print-terms-and-conditions';
 	const paymentFrequencyName =
 		props.paymentFrequency === 'year' ? 'annual' : 'monthly';
+	const productName =
+		props.countryGroupId === 'UnitedStates'
+			? productNameUSTier3
```
Comment (jamesgorrie) [inline support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx:46]: The convention i the codebase it to name things as you say them e.g. `TierThreeUSProductName`. Don't mind, but as previously stated keeping with convention makes guessing easier down the line.

File: support-frontend/assets/pages/[countryGroupId]/checkout.tsx [hunk 4]
```diff
@@ -1652,6 +1655,24 @@ function CheckoutComponent({
...
+												? 'MONTHLY'
+												: productFields.billingPeriod === 'Annual'
+												? 'ANNUAL'
+												: 'ONE_OFF'
+										}
+										currency={currencyKey}
+										amount={price}
+										amountIsAboveThreshold={
+											productDescription.label === 'All-access digital' ||
+											productDescription.label === 'Digital + print'
```
Comment (jamesgorrie) [inline support-frontend/assets/pages/[countryGroupId]/checkout.tsx:1671]: As these are likely to change, should we use the `productKey`?

File: support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx [hunk 5]
```diff
@@ -222,8 +244,92 @@ export function PaymentTsAndCs({
...
 	);
 }
+
+export function SummaryTsAndCs({
+	contributionType,
+	countryGroupId,
+	currency,
+	amount,
+	amountIsAboveThreshold,
+	productNameAboveThreshold,
```
Comment (jamesgorrie) [inline support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx:259]: Using `productKey` would make this less brittle to future label changes.

File: support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx [hunk 6]
```diff
@@ -222,8 +244,92 @@ export function PaymentTsAndCs({
...
+}: PaymentTsAndCsProps): JSX.Element {
+	const inTier3 =
+		productNameAboveThreshold === 'Digital + print' && amountIsAboveThreshold;
+	const inTier2 =
+		productNameAboveThreshold === 'All-access digital' &&
+		amountIsAboveThreshold;
+	const inTier1 =
+		productNameAboveThreshold === 'Support' || !(inTier2 || inTier3);
+
+	const amountCopy = isNaN(amount)
```
Comment (jamesgorrie) [inline support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx:269]: When is `amount` `NaN`?

**Review by jamesgorrie — COMMENTED** [review]
I really like the idea of just breaking the Ts and Cs and other copy out for each product. Could be worth thinking if we add it to the `productCatalogDescription`?

I have a couple of questions on using keys instead of labels in conditionals is the only reason I haven't 🍏 this.

File: support-frontend/assets/pages/[countryGroupId]/checkout.tsx (hunk 4 above)
Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/[countryGroupId]/checkout.tsx:1671]: Ah. yes, better to use that, changed

File: support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx (hunk 5 above)
Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx]: Have used `productKey` throughout this file now, unpacking the associated name at the end, agree more robust that way.

File: support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx (hunk 6 above)
Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx:261]: Agree, removed.

File: support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx (hunk 1 above)
Comment (paul-daniel-dempsey) [reply support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx:16]: Agree, changed.

File: support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx (hunk 3 above)
Comment (paul-daniel-dempsey) [reply support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx]: Ok, will try to keep with convention, changed

File: support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx (hunk 2 above)
Comment (paul-daniel-dempsey) [reply support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx]: moved this across to `productCatalog` file

**Review by jamesgorrie — APPROVED** [review]
Looks good - I still think we should try standardise around `TierThree` instead of `Tier3` though.

File: support-frontend/assets/components/subscriptionCheckouts/tierThreeTerms.tsx [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+					{ManageMyAccountLink} or see our {productName}{' '}
+					{termsLink('Terms', tierThreeTermsLink)}.
+				</p>
+				<p>
+					By proceeding, you are agreeing to the {productName}{' '}
+					{termsLink('Terms', tierThreeTermsLink)}.
+				</p>
+			</>
+		);
+	};
```
Comment (jamesgorrie) [inline support-frontend/assets/components/subscriptionCheckouts/tierThreeTerms.tsx:64]: Could you explain the benefit of moving this out into it's own method?

File: support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx [hunk 8]
```diff
@@ -211,19 +231,94 @@ export function PaymentTsAndCs({
...
+export function SummaryTsAndCs({
+	contributionType,
+	countryGroupId,
+	currency,
+	amount,
+	amountIsAboveThreshold,
+	productKey,
+	cssOverrides,
+}: PaymentTsAndCsProps): JSX.Element {
+	const inTier3 = productKey === 'TierThree' && amountIsAboveThreshold;
```
Comment (jamesgorrie) [inline support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx:255]: Why do we need `amountIsAboveThreshold` if the amount is defined by the product API

File: support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx [hunk 9]
```diff
@@ -211,19 +231,94 @@ export function PaymentTsAndCs({
...
+	};
+
+	const copyTier3 = (
+		contributionType: ContributionType,
+		productKey: ProductKey,
+	) => {
+		return (
+			<>
+				<div>
+					The {productCatalogDescription[productKey].label} subscriptions will
```
Comment (jamesgorrie) [inline support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx:304]: Nice to see this becoming the source of truth!

**Review by jamesgorrie — APPROVED** [review]
this is quite fiddly work - good work on getting it out clearly.

File: support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx (hunk 9 above)
Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx:304]: Thanks, great recommend to pass `productKey` down instead

File: support-frontend/assets/components/subscriptionCheckouts/tierThreeTerms.tsx (hunk 7 above)
Comment (paul-daniel-dempsey) [reply support-frontend/assets/components/subscriptionCheckouts/tierThreeTerms.tsx]: Your right, not needed. Originally we had a US specific copy version we switched between but the copy back to its original state yesterday. I did not clear it down far enough. Moved this markup back down.

File: support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx (hunk 8 above)
Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx]: Agree not needed for Summary component, removed

**Pull request merged**